docs/content/try-out.md

```diff
@@ -8,7 +8,7 @@ menu: shortcuts
 
 To quickly see Athens in action, follow these steps:
 
-First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/),
+First, make sure you have [Go 1.11 or later installed](https://gophersource.com/setup/),
```

Comment:
Actually we should say `Go 1.12` here because we only guarantee support from 1.12 on due to the checksum mismatches and the ability to do `go get` outside of a module repository.

Reply:
OK!